docs/guide/save.rst

```diff
@@ -46,14 +50,20 @@ can force Willow to always save a "progressive" JPEG file by setting the
     with open('progressive.jpg', 'wb') as f:
         i.save_as_jpeg(f, progressive=True)
 
-Lossless WebP
+Lossless Avif, Heic and WebP
 -----------------
 
-You can encode the image to WebP without any loss by setting the
+You can encode the image to Avif, Heic (only pillow) and WebP without any loss by setting the
```

Comment:
```suggestion
You can encode the image to AVIF, HEIC (Pillow-only) and WebP without any loss by setting the
```

note that ImageMagick (thus Wand) supports HEIC via libheif since version 7.1.0. Opened #119

Reply:
we can change the docs further in that PR, would you mind if I take up that issue as well?

Reply:
Happy for you to take it on, once this and the Wagtail continuation are done